content/blog/2023/knight_odds.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Play with knight odds against Lc0 on lichess"
+author: "Naphthalin"
+published: 2023-11-07
```

Comment:
I guess date should be changed to today?

Reply:
Done